perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -594,6 +594,19 @@ sub waste_container_actions {
     };
 }
 
+sub waste_recently_expired {
+    my ($self, $schedule) = @_;
+
+    my ($y, $m, $d) = ( $schedule->{end_date} =~ /(\d+)-(\d+)-(\d+)/);
+    my $end = DateTime->new( year => $y, month => $m, day => $d)->set_time_zone(FixMyStreet->local_time_zone);
+    my $now = DateTime->now->set_time_zone(FixMyStreet->local_time_zone);
+    if ( $now > $end ) {
```

Comment:
Is the end date inclusive? (I assumed it was.) So this is true on the day a schedule expires, should it be?

Reply:
Could perhaps waste_sub_overdue (which seems to have a fix for this in this commit), be given a day or week argument, then it returns true if more than that overdue (and so 0 by default?)